components/cam/src/physics/cam/shoc.F90

```diff
@@ -1173,6 +1241,7 @@ subroutine diag_third_shoc_moments(&
       isosqrt=iso**2
       buoy_sgs2=isosqrt*brunt_zi(i,k)
       bet2=ggr/thetal_zi(i,k)
+ 
```

Comment:
Nitpick: I think we can move `kb` and `kc` computation just after `do k=2,nlev` as we do not need to recompute `kb` and `kc` for each `i`.